constance/admin.py

```diff
@@ -272,11 +272,15 @@ def changelist_view(self, request, extra_context=None):
                         config_values.append(
                             self.get_config_value(name, options, form, initial)
                         )
-
-                context['fieldsets'].append({
+                fieldset_context = {
                     'title': fieldset_title,
                     'config_values': config_values
-                })
+                }
+
+                if (settings.CONFIG_COLLPASED_FIELDSETS and 
```

Comment:
It would be nice to remove trailing whitespace
```suggestion
                if (settings.CONFIG_COLLPASED_FIELDSETS and
```